e2e/e2etests/test_message_passing_zevm.go

```diff
@@ -0,0 +1,332 @@
+package e2etests
+
+import (
+	"fmt"
+	"math/big"
+
+	ethcommon "github.com/ethereum/go-ethereum/common"
+	"github.com/zeta-chain/zetacore/e2e/contracts/testdapp"
+	"github.com/zeta-chain/zetacore/e2e/runner"
+	"github.com/zeta-chain/zetacore/e2e/utils"
+	cctxtypes "github.com/zeta-chain/zetacore/x/crosschain/types"
+)
+
+func TestMessagePassingEVMtoZEVM(r *runner.E2ERunner, args []string) {
+	if len(args) != 1 {
+		panic("TestMessagePassing requires exactly one argument for the amount.")
+	}
+	amount, ok := big.NewInt(0).SetString(args[0], 10)
+	if !ok {
+		panic("Invalid amount specified for TestMessagePassing.")
+	}
+
+	// Set destination details
+	zEVMChainID, err := r.ZEVMClient.ChainID(r.Ctx)
+	if err != nil {
+		panic(err)
+	}
+	destinationAddress := r.ZevmTestDAppAddr
```

Comment:
```suggestion
	}

	destinationAddress := r.ZevmTestDAppAddr
```
Just a general comment for here and following. I think it is more visual to add empty line after error handling

Reply:
In my opinion, grouping sections of code might be a good idea in this case and separating these groups using line breaks. Adding a line after each error might make the code look more fragmented.